ucte/ucte-network/src/main/java/com/powsybl/ucte/network/ext/UcteNetworkExt.java

```diff
@@ -123,21 +123,35 @@ public UcteRegulation getRegulation(UcteElementId transfoId) {
         }
 
         // in the same substation...
-        addEdgeBetweenSameGeographicalSpotNodes(network, graph);
+        addEdgeBetweenSameGeographicalSpotNodes(network, Optional.empty(), graph);
```

Comment:
It is really not recomanded to use an optiona as a parameter, only for return type.